code/src/App.js

```diff
@@ -1,9 +1,76 @@
-import React from 'react'
+import React, { useEffect, useState } from "react";
+import ThoughtForm from "./components/ThoughtForm";
+import ThoughtItem from "./components/ThoughtItem";
+import LoadingItem from  './components/Loading';
+import { API_URL, LIKES_URL } from "./utils/urls";
+import { Header } from 'Header'
+// import { Footer } from 'Footer'
 
 export const App = () => {
-  return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+	const [thoughts, setThoughts] = useState([]);
+	const [newThought, setNewThought] = useState("");
+  const [loading, setLoading] = useState(false);
```

Comment:
Nice job keeping all your state up at the top, just don't forget to use prettier or check indentation just before you push